roles/rhods_test_jupyterlab/tasks/main.yml

```diff
@@ -176,6 +176,27 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Update default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+    NOTEBOOK_RAM: '{{ rhods_test_jupyterlab_notebook_memory }}'
+    NOTEBOOK_CPU: '{{ rhods_test_jupyterlab_notebook_cpu }}'
+
+  command: |
```

Comment:
won't work :) this is a `shell`, not a `command` (`command` is a plain `exec(3)`, while `shell` is `bash -c ....`)

Reply:
Adapted, but see below